.github/workflows/CI.yml

```diff
@@ -4,24 +4,22 @@ on: [push]
 
 jobs:
   test:
-
+    timeout-minutes: 15
```

Comment:
question: why have you decided to add timeout?

Reply:
Due to the fact that by default Github actions have 360 hours set for timeout then in case CI is written with an error or something goes wrong and it blocks on an infinity loop for some reason, we will block resources for 6 hours. Setting it to 15 minutes is a contractual matter, but it is better to shorten this timeout for such a case